apps/storefront/.env.example

```
@@ -0,0 +1,7 @@
+
+DATABASE_URL=postgresql://user:password@localhost:5432/dukkani
+NEXT_PUBLIC_CORS_ORIGIN=http://localhost:3002
+TELEGRAM_BOT_NAME=
+NEXT_PUBLIC_ALLOWED_ORIGIN=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_ALLOWED_ORIGIN key should go before the NEXT_PUBLIC_CORS_ORIGIN key

(UnorderedKey)